Composer/packages/electron-server/src/utility/machineId.ts

```diff
@@ -13,17 +13,20 @@ import { readTextFileSync, writeJsonFileSync } from './fs';
 
 export const persistedFilePath = path.join(app.getPath('userData'), 'persisted.json');
 
+function truncate(str: string): string {
```

Comment:
Arrow function

Reply:
I don't see what that would get us here. This isn't inside a component that gets re-rendered, this isn't exported anywhere, it's just a simple utility function that belongs to this one file.

Reply:
The other functions in this file use arrow functions. Best to be consistent. More of a nit.